genesis/worlds/small.json

```diff
@@ -0,0 +1,36 @@
+{
+    "from": {
+        "x": -10,
+        "y": -10,
+        "z": -10
+    },
+    "to": {
+        "x": 10,
+        "y": 10,
+        "z": 10
+    },
+    "asteroid_adjs": [
+        "Large",
+        "Medium",
+        "Small",
+        "Dark",
+        "Bright"
+    ],
+    "asteroid_colors": [
+        "#c0c0c0",
+        "#c0c0c0",
+        "#c0c0c0",
+        "#c0c0c0",
+        "#c0c0c0"
+    ],
+    "starbase_color": "#eff7de",
+    "asteroids": 10,
+    "shard_name": "mainworld",
```

Comment:
let's not use the same shard name for the two different shard size/types. Probably not a big issue, but it might help avoid confusion during testing :) Maybe just call this one `smallworld` instead of `mainworld`

Reply:
Good point, will fix